### PR TITLE
[WFLY-10288] Remove remaining Java EE 8 Preview information

### PR DESCRIPTION
**Pull request by jamezp**
https://issues.jboss.org/browse/WFLY-10288

The first commit just updates the documentation removing the reference to Java EE 8 preview mode.

The second commit removes the JAX-WS Java EE 7 support. It looks like this one got missed somehow.

The third commit moves the `BeanValidationEE8TestCase` to the integration tests.

The forth commit cleans up the rest of the Java EE 8 tech preview resources.

### CODE REVIEW

File: docs/pom.xml [hunk 1]
```diff
@@ -48,6 +48,10 @@
                     <groupId>org.asciidoctor</groupId>
                     <artifactId>asciidoctor-maven-plugin</artifactId>
                     <configuration>
+                        <attributes>
+                            <javaee_version>8</javaee_version>
+                            <wildflyversion>${verifier.product.release.version}</wildflyversion>
```
Comment (bstansberry) [inline docs/pom.xml]: This is kind of a hijack of something meant and documented for a different purpose. If we want different uses we should probably rework the property and clarify it's use.

For the verifier use it's major.minor while wildflyversion in the past was just major. So there's a bit of a mismatch there.

Comment (bstansberry) [reply docs/pom.xml]: Any change to the properties should be discussed with Yeray though, as these are important for eap-ization and any changes will need to be accounted for there.

Comment (jamezp) [reply docs/pom.xml]: Yeah. It's on my documentation todo list to change several constants to attributes.

Sure yeah. We don't have to use that version as it was set to just "13" before. I just figured the less places we need to make changes the better. I kind of wonder if these docs should just be in their own repository anyway.

**Comment by scottmarlow** [comment]
Would it make sense to keep the BeanValidationEE8TestCase test?  Perhaps move it to testsuite/integration/basic?  I'm not sure if there is anything worth keeping in it but thought I would ask.

**Review by asoldano — APPROVED** [review]
Thanks for the jaxws module fix

**Comment by jamezp** [comment]
We could keep the test. I just deleted the `ee8-temp` module. Since it was in that module I guess it fell victim :)

**Comment by rsvoboda** [comment]
@mjurc - keep the BeanValidationEE8TestCase ?

**Comment by mjurc** [comment]
Please move the `BeanValidationEE8TestCase` to `testsuite/integration/basic` module. It seems to be a bit more thorough than our standing `BeanValidationTestCase`.

**Review by kabir — CHANGES_REQUESTED** [review]
In addition to the above comments, please also remove the full-feature-pack-licenses-addendum.html and servlet-feature-pack-licenses-addendum.html files as I think we should now have removed all the remaining hardcoded versioned jars (ie the ones not coming from the pom)?

**Comment by jamezp** [comment]
Thanks @kabir I missed those and they are now removed.

@mjurc @rsvoboda I added a commit that moves the `BeanValidationEE8TestCase` to the integration tests.

**Review by rsvoboda — APPROVED** [review]
I have no further comments?

**Review by mjurc — APPROVED** [review]

File: docs/pom.xml (hunk 1 above)
Comment (kabir) [reply docs/pom.xml]: @jamezp I think a product.docs.server.version in the root pom next to verifier.product.release.version will make this cleaner

Comment (jamezp) [reply docs/pom.xml]: Done.

**Comment by jamezp** [comment]
@scottmarlow It was moved in commit https://github.com/wildfly/wildfly/pull/11424/commits/5ddf0bfc13382cb580cd1c4b8f100467c34fc972. It's the third commit.

**Pull request merged**